packages/core/src/tools/error.ts

```diff
@@ -65,3 +74,26 @@ export function toStackTraceString(stack: StackTrace) {
 export function formatErrorMessage(stack: StackTrace) {
   return `${stack.name || 'Error'}: ${stack.message!}`
 }
+
+/** Has to be called at the utmost position of the RUM call stack */
+export function createHandlingStackTrace(): StackTrace {
+  const instrumentationCallDepth = 2
+  const error = new Error()
+  let stackTrace: StackTrace
+
+  // IE needs to throw the error to fill in the stack trace
+  if (!error.stack) {
+    try {
+      throw error
+    } catch (e) {
+      noop()
+    }
+  }
+
+  callMonitored(() => {
+    stackTrace = computeStackTrace(error)
+    stackTrace.stack = stackTrace.stack.slice(instrumentationCallDepth)
+  })
```

Comment:
why do we need to wrap this part in a `callMonitored`?

Reply:
We need to wrap it in `callMonitored` because `createHandlingStackTrace` is call outside of them in order for `const error = new Error()` to be at the upmost point of the RUM callstack.
https://github.com/DataDog/browser-sdk/blob/c7d3145240054d9dd115c205912e84c3f59f1339/packages/rum-core/src/boot/rumPublicApi.ts#L157-L159
and 
https://github.com/DataDog/browser-sdk/blob/c7d3145240054d9dd115c205912e84c3f59f1339/packages/core/src/domain/automaticErrorCollection.ts#L38-L40

Reply:
OK makes sense, it could be nice to mention that in the function description